dex/src/main/scala/com/wavesplatform/dex/cli/WavesDexCli.scala

```diff
@@ -268,6 +296,64 @@ object WavesDexCli extends ScoptImplicits {
                 case Right(_) =>
                 case Left(error) => println(error); forceStopApplication(MatcherStateCheckingFailedError)
               }
+
+            case Command.MakeOrderbookSnapshots =>
+              cli.log(
+                s"""
+                   |Passed arguments:
+                   |  DEX config path : ${args.configPath}
+                   |  DEX REST API    : ${args.dexRestApi}
+                   |  Timeout         : ${args.timeout}
+                   |Running in background
+                   |""".stripMargin
+              )
+
+              val apiUrl =
+                if (args.dexRestApi.isEmpty) {
+                  val matcherSettings =
+                    ConfigSource.fromConfig(loadConfig(parseFile(new File(args.configPath)))).at("waves.dex").loadOrThrow[MatcherSettings]
+                  s"${matcherSettings.restApi.address}:${matcherSettings.restApi.port}"
+                } else args.dexRestApi
+
+              def get(urlPart: String, key: String): Int = {
+                println(s"Getting: $urlPart")
+                basicRequest
+                  .get(uri"$apiUrl/matcher/debug/$urlPart")
+                  .headers(Map("X-API-KEY" -> key))
+                  .send()
+                  .body match {
+                  case Right(x) => x.toInt
```

Comment:
This match statement should be refactored

Reply:
refactored